### ISSUE TITLE
Concern for use of "must" in notes.

### ISSUE THREAD

**Issue by bruce-usab** [w3c/wcag#4432]
In the WCAG2ICT call today, we discussed that use of "must" in a note is problematic.

> **Note:**  Since any content that does not meet this success criterion can interfere with a user's ability to use the whole page, all content on the web page (whether it is used to meet other success criteria or not) **must** meet this success criterion. See [Conformance Requirement 5: Non-Interference](https://www.w3.org/TR/WCAG22/#cc5).

Suggestion on call is to replace "must" with "would be necessary" or similar.

**Comment by patrickhlauke** [comment]
While I understand the desire not to have MUST/SHOULD in non-normative prose, as that would introduce new normative requirements by the backdoor, I think in this case this is used as a way to signpost an already existing normative requirement coming from that conformance requirement 5?

**Comment by maryjom** [comment]
The EN 301 549 is not allowed to have normative words in non-normative statements, and I thought that W3C standards were the same (not sure, but it would make sense). The WCAG2ICT TF discussed today - "would need to" might be sufficiently imperative without saying "must".

**Comment by bruce-usab** [comment]
I agree that this is problematic for WCAG2ICT but merely editorial for WCAG 2.

Yes, the WCAG 2 citation to the conformance requirements supports the use of "must".   WCAG2ICT has [comments on conformance](https://www.w3.org/TR/wcag2ict/#comments-on-conformance) but that's different.

WCAG2ICT also has a grammar issue in that the [word substitution](https://www.w3.org/TR/wcag2ict/#applying-sc-1-4-2-audio-control-to-non-web-documents-and-software) differs between the criterion text and the note.  

I could not make a simple word substitution for "must" work.  How about:

> **Note:**  Since any content that does not meet this success criterion can interfere with a user's ability to use the whole page, it is necessary that all content on the web page (whether it is used to meet other success criteria or not) meet this success criterion.

**Comment by maryjom** [comment]
Similar to a couple of proposals for this note in the [WCAG2ICT issue 680](https://github.com/w3c/wcag2ict/issues/680), here's a couple more variations of @bruce-usab's suggestion above with verbiage changes bolded to help with reviewing in the context of this issue. It wouldn't be bolded when incorporated into WCAG.

>**Note:** Since any content that does not meet this success criterion can interfere with a user's ability to use the whole page, **it would be necessary for** all content on the web page (whether it is used to meet other success criteria or not) to meet this success criterion. See [Conformance Requirement 5: Non-Interference](https://www.w3.org/TR/WCAG22/#cc5).

OR

>**Note:** Since any content that does not meet this success criterion can interfere with a user's ability to use the whole page, all content on the web page (whether or not it is used to meet other success criteria) **would need to meet** this success criterion. See [Conformance Requirement 5: Non-Interference](https://www.w3.org/TR/WCAG22/#cc5).

**Comment by mbgower** [comment]
**Draft Official Working Group response**
The  WCAG note appearing in 1.4.2., 2.1.2, 2.1.2, and 2.3.1 that contains this "must" wording refers directly to 5.2.5 which states (in normative text):
> If [technologies ](https://www.w3.org/TR/WCAG22/#dfn-technologies)are used in a way that is not [accessibility supported](https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported), or if they are used in a non-conforming way, then they do not block the ability of users to access the rest of the page. In addition, the [web page](https://www.w3.org/TR/WCAG22/#dfn-web-page-s) as a whole continues to meet the conformance requirements under each of the following conditions:

> 1. when any technology that is not [relied upon](https://www.w3.org/TR/WCAG22/#dfn-relied-upon) is turned on in a user agent,
> 2. when any technology that is not relied upon is turned off in a user agent, and
when any technology that is not relied upon is not supported by a user agent
> 3. In addition, the following success criteria apply to all content on the page, including content that is not otherwise relied upon to meet conformance, because failure to meet them could interfere with any use of the page:

>- 1.4.2 - Audio Control,
>- 2.1.2 - No Keyboard Trap,
>- 2.3.1 - Three Flashes or Below Threshold, and
>- 2.2.2 - Pause, Stop, Hide.

So the use of "must" is appropriate in this case; it is not adding a new requirement in a note, but restating an existing requirement. The Working Group does not believe a change to the note is warranted in this case. 

The WCAG2ICT Task Force is welcome to reword this note in their own TF guidance to provide a similar message without using the word "must".

**Comment by Dfghj887533** [comment]
-

**Comment by bruce-usab** [comment]
Discussed on backlog call 6/6 and TF concurrence not to make a change.  We _might_ revisit after WCAG2ICT and EN 301 549.

**Comment by mbgower** [comment]
Based on a 2-week review period by the working group, and receiving support for the proposed WG response, I had updated the draft response to be the official response.
